xcode/Subconscious/Shared/Components/Notebook/Notebook.swift

```diff
@@ -115,7 +115,7 @@ enum NotebookAction {
     /// Set the count of existing entries
     case setEntryCount(Int)
     /// Fail to get count of existing entries
-    case failEntryCount(Error)
```

Comment:
Necessary to make NotebookAction Hashable, and we need it to be Hashable so that we can test for value equality in the tests.